components/automate-workflow-server/habitat/default.toml

```diff
@@ -8,8 +8,8 @@
 api_port = 9611
 audit_max_events = 100
 ca_cert_chain_depth = 2
-db_pool_init_count = 20
-db_pool_max_count = 100
```

Comment:
do we know why max was so high here? was there no queue?

Reply:
I think we've been copy/pasta'ing the defaults rather than thinking about it.

https://github.com/chef/chef-server/blob/master/src/oc_erchef/habitat/default.toml#L61